### ISSUE TITLE
AMA intake | Review: select Board lane (NOD)

### ISSUE THREAD

**Issue by abbyraskinUSDS** [department-of-veterans-affairs/caseflow#5169]
This is the page where Board intake users can enter the Receipt Date, and the specific Board lane the Veteran has requested on their AMA appeal.

## Acceptance Criteria
- User navigates to this page by selecting a Notice of Disagreement on the first step, and submitting a valid search on the previous page
- Receipt date is not in the future
- Receipt Date is required
- "Cancel intake" button launches same modal as RAMP (for now)
- Selecting "Continue" button navigates user to the Finish processing page once it is developed 

## Mocks
![image](https://user-images.githubusercontent.com/4042181/39486148-21c034be-4d49-11e8-9d5c-b17433625510.png)

**Comment by abbyraskinUSDS** [comment]
Since the "more than one review option" is a bit of an edge case, I'd propose we hold off on developing it until we're closer to AMA (i.e., not for pilot).

**Comment by abbyraskinUSDS** [comment]
Follow up with Mark on his data model spec

**Comment by leikkisa** [comment]
Just going to get started even though I know we're waiting on the appeals table.

**Comment by leahbannon** [comment]
I ran into an issue on this:
* enter date and review option on Review Request page, click Continue
* get totally blank page, possibly also this error https://dsva.slack.com/archives/C2TB5MH6F/p1526330584000201
* hit refresh, get list of issues contentions to check off
* cancel out of intake
* enter vet's ID again, get "TEST ONE already started processing this form" error
(I should be able to restart this after canceling it.)

**Comment by leikkisa** [comment]
I think if it was the "no ratings" issue, then you wouldn't get the list of issues after refreshing. I am having issues with my environment right now so will be tough for me to replicate it, sometimes also having blank page issues, and my AMA feature toggle isn't working :-/.  However, I'm looking at the code for another issue so I'll see if I notice anything in the code.

**Comment by leikkisa** [comment]
Shane helped me fix my bug so I can see AMA stuff again, I'll take a look.

**Comment by shanear** [comment]
@leahbannon your bug was due to another issue that should have been fixed

**Comment by leahbannon** [comment]
this looks good
